test/filters/unit_tests.filter

```diff
@@ -616,6 +616,12 @@
 # We disable HATS
 -HatsHandlerTest.HandleSecurityPageHatsRequestPassesArgumentsToHatsService
 
+# These tails fail because we add a new quick action to download bubble to delete local file.
+-DownloadBubbleRowViewInfoTest.InProgressOrCompletedInfo
+#-DownloadBubbleRowViewTest.InputProtectorDeniesClicks
+#-DownloadBubbleRowViewTest.OnlyEnabledQuickActionsVisible
+#-DownloadBubbleRowViewTest.UpdateTimeFromCompletedDownload
```

Comment:
leftover?

and we have same list filtered at `unit_tests-macos.filter`.
If we want to enable these tests, maybe need to touch that filter file also.

Reply:
> If we want to enable these tests, maybe need to touch that filter file also. Or they can't run on macOS?

Fixed here https://github.com/brave/brave-core/pull/30120